docs/fsharp-cheatsheet.md

```diff
@@ -411,17 +459,20 @@ Another way of implementing interfaces is to use *object expressions*.
 *Single-case active patterns*:
 
     // Basic
-    let (|EmailDomain|) (email: string) =
-        let parts = email.Split '@'
-        parts[1]
-    let (EmailDomain emailDomain) = "yennefer@aretuza.org" // emailDomain = 'aretuza.org'
+    let (|EmailDomain|) email =
+        let match' = Regex.Match(email, "@(.*)$")
+        if match'.Success
+        then match'.Groups[1].ToString()
```

Comment:
I dont like the then on a new line (I'd join the if and then lines)
but it probably is good to show it this way pedagogically